AjobK/Athena
# 52 templates

closes #52 

**What is it supposed to do**
It makes sure we all get some structure in our issues en pull requests

**How can we test your branch**
It is hard to test what you can do is look at the template and check for mistakes in grammar. If you really want to test the templates you need to test it in your own repo. I did test it in my own repo to make sure it works.

**What broweser have been checked**
It can't be checked in a browser because it is not in Seaqull.
